pyenv-win/bin/pyenv.bat

```diff
@@ -2,20 +2,20 @@
 setlocal
 chcp 1250 >nul
 
-set "pyenv=cscript //nologo "%~dp0..\libexec\pyenv.vbs""
+set "__pyenv_pyenv=cscript //nologo "%~dp0..\libexec\pyenv.vbs""
```

Comment:
Why prefix all variables with `__pyenv_` ?

Reply:
All I can say in my defence is is must have been around 3-4am... I would have sworn my Oh My Posh python segment showed that command, as the python version at some point, but I can't reproduce it.